### PR TITLE
Clotheshop, Accessories, Status, Sit Modules (Do not merge)

### PR DESCRIPTION
**Pull request by BrakingESX247**
## Proposed Changes

    - Initial commit for clotheshop module. Still needs work, but It should be a good start.
    - Latest code applied to accessories module.
    - A lot of additions to utility module.
    - Sit module (75% done).
    - Animation module (base).

### CODE REVIEW

**Comment by izio38** [comment]
Wow, awesome contribution, thanks for the PR !

Do you want us to review/test this or you havn't finished yet ?

**Comment by BrakingESX247** [comment]
I think I am done yes. I just need a review from you guys and then I can change/merge it.

File: modules/__base__/clotheshop/client/module.lua [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+      local key = 'clotheshop:' .. k .. ':' .. i
+
+      Interact.Register({
+        name = key,
+        type = 'marker',
+        distance = module.Config.DrawDistance,
+        radius = 2.0,
+        pos = v[i],
+        size = module.Config.Size.z,
```
Comment (Dislaik) [inline modules/__base__/clotheshop/client/module.lua:39]:
```suggestion
        size = module.Config.MarkerSize.z,
```

Comment (ArkSeyonet) [reply modules/__base__/clotheshop/client/module.lua:39]: I will look tonight and get back with you on the review

**Comment by ArkSeyonet** [comment]
> I think I am done yes. I just need a review from you guys and then I can change/merge it.

I will look tonight sometime and try to review

File: modules/__base__/sit/client/main.lua [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+--   If you redistribute this software, you must link to ORIGINAL repository at https://github.com/ESX-Org/es_extended
+--   This copyright should appear in every part of the project code
+
+module.init()
+
+ESX.SetInterval(1, function()
+		local playerPed = GetPlayerPed(-1)
+
+		if module.sitting and not IsPedUsingScenario(playerPed, currentScenario) then
+			module.wakeup()
```
Comment (abdulkadiraktas) [inline modules/__base__/sit/client/main.lua:19]: not necessary

File: modules/__base__/sit/client/main.lua [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+		end
+
+		if IsControlJustPressed(0, 38) and IsControlPressed(0, 21) and IsInputDisabled(0) and IsPedOnFoot(playerPed) then
+			if module.sitting then
+				module.wakeup()
+			else
+				local object, distance = utils.game.getClosestObject(Config.Interactables)
```
Comment (abdulkadiraktas) [inline modules/__base__/sit/client/main.lua:26]: Not Working.

local coordss = GetEntityCoords(playerPed)
			for i,v in ipairs(module.Config.Interactables) do
				--object, distance = utils.game.getClosestObject(coordss,v)				
				object = GetClosestObjectOfType(coordss.x, coordss.y, coordss.z, 1.4, GetHashKey(v), false, false, false)
				distance = 1.4
				if object > 0 then
					break
				end 
			end

File: modules/__base__/sit/client/module.lua [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+  end)
+end
+
+module.wakeup = function()
+	local playerPed = GetPlayerPed(-1)
+	ClearPedTasks(playerPed)
+
+	module.sitting = false
+
+	SetEntityCoords(playerPed, module.lastPos)
```
Comment (abdulkadiraktas) [inline modules/__base__/sit/client/module.lua:60]: not necessary

module.wakeup = function()
	local playerPed = GetPlayerPed(-1)  
  ClearPedTasks(playerPed)
	module.sitting = false
	--SetEntityCoords(playerPed, module.lastPos)
	emitServer('esx_sit:leavePlace', module.currentSitCoords)
  module.currentSitCoords, module.currentScenario = nil, nil
	FreezeEntityPosition(playerPed, false)
	ClearPedTasks(playerPed)
end

File: modules/__base__/sit/client/main.lua (hunk 2 above)
Comment (BrakingESX247) [reply modules/__base__/sit/client/main.lua:27]: > Sitting module is work now.

Thank you for the commit <3

**Comment by IqFiskMos** [comment]
Would be nice if all labels for animations would have locales instead of being in config and in french. Really nice job done overall

**Pull request closed without merging**